### PR TITLE
docs: update library descriptions

### PR DESCRIPTION
**Pull request by mattrosno**
Closes https://github.com/carbon-design-system/carbon-platform/issues/1109

### Related Ticket(s)

N/A

### Description

- Updates indexed library descriptions, per design spec: https://airtable.com/shrdwElZNyJdGtZkS/tblE2Y75BfY0Z0T2O

### Changelog

**New**

- N/A

**Changed**

- Indexed library description
- Fixed indexed thumbnail path, found here: https://github.com/carbon-design-system/carbon-platform/pull/1127#issuecomment-1211433470
- Made sure we're using `’` instead of `'` in all descriptions

**Removed**

- N/A

**Tests**

With `INDEX_ALL=1` and only these libraries...

Updated library descriptions:
image

Lead space search SVG works:
image

### CODE REVIEW

**Review by alisonjoseph — APPROVED** [review]

**Review by ariellalgilmore — APPROVED** [review]
LGTM

File: packages/services/carbon.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 library:
   id: ibmdotcom-services
   name: IBM.com Services
-  description: IBM.com ES6 service classes
+  description: Apply JavaScript ES6 service classes that adhere to IBM's web standards.
```
Comment (francinelucca) [inline packages/services/carbon.yml:5]: is this final? airtable says in progress

File: packages/styles/carbon.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 library:
   id: ibmdotcom-styles
   name: IBM.com Styles
-  description: IBM.com framework agnostic component styles
+  description: Apply framework-agnostic styles to any IBM.com component.
```
Comment (francinelucca) [inline packages/styles/carbon.yml:5]: is this final? airtable says in progress

File: packages/utilities/carbon.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 library:
   id: ibmdotcom-utilities
   name: IBM.com Utilities
-  description: IBM.com ES6 utility classes
+  description: Apply JavaScript ES6 utility classes that adhere to IBM's web standards.
```
Comment (francinelucca) [inline packages/utilities/carbon.yml:5]: is this final? airtable says in progress

**Comment by mattrosno** [comment]
@aubrey-oneal can you confirm that library descriptions for Services, Utilities, and Styles above are good?

File: packages/utilities/carbon.yml (hunk 3 above)
Comment (aubrey-oneal) [reply packages/utilities/carbon.yml:5]: yup, Matt finalized the description

File: packages/styles/carbon.yml (hunk 2 above)
Comment (aubrey-oneal) [reply packages/styles/carbon.yml:5]: yup, Matt finalized the description

**Review by francinelucca — APPROVED** [review]

**Comment by mattrosno** [comment]
@aubrey-oneal I updated to use `’` in the descriptions.

**Comment by mattrosno** [comment]
Hi @kennylam can I get a review? Thanks!

**Review by andreancardona — APPROVED** [review]

**Review by kennylam — APPROVED** [review]
LGTM thanks @mattrosno!

**Pull request merged**